float32 encFL
float32 encFR
float32 encBL
float32 encBR
float32 p
float32 q
float32 r
float32 ax
float32 ay
float32 az
uint8 stopFlag
uint8 turnFlag
